site/content/docs/5.3/examples/form/index.html

```diff
@@ -217,7 +217,7 @@ <h1 class="display-1">Become a reseller</h1>
   <h2 class="visually-hidden">Sitemap and information</h2>
   <div class="container-xxl footer-terms">
     <ul class="navbar-nav gap-md-3">
-      <li class="fw-bold">© Orange 2022</li>
+      <li class="fw-bold">© Orange 2023</li>
```

Comment:
Cant' we use something like `{{< year >}}` or `{{ now.Format "2006" }}` ? As we do in `.md` and in `/shortcodes`?

Reply:
Nice enhancement ✨ Seems to work well with `{{< year >}}`. Fixed in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1842/commits/a003c0b09a482f5e442c1968b892334e9f9c9728.